examples/step-72/step-72.cc

```diff
@@ -416,27 +430,24 @@ namespace Step72
                                        system_rhs);
   }
 
-  // The second method of assembly is going to use automatic differentiation
-  // to compute the linearization of the residual vector. More precisely,
-  // given the discrete residual associated with each degree of freedom $I$
-  // @f[
-  //   F^{I}(u) \dealcoloneq
-  //   \int\limits_{\Omega}\nabla v^{I} \cdot \left[ \frac{1}{\sqrt{1+|\nabla u|^{2}}} \nabla u \right] \, dV ,
-  // @f]
-  // and its first order Taylor expansion for each finite element $e$ with 
-  // respect to each degree of freedom $J$
+  // @sect5{Assembly via differentiation of the residual vector}
+
+  // As outlined in the introduction, what we need to do for this
+  // second approach is implement the local contributions $F(U)^K$
+  // from cell $K$ to the residual vector, and then let the
+  // autodifferentiation machinery deal with how to compute the
```

Comment:
```suggestion
  // AD machinery deal with how to compute the
```
(or automatic differentiation)